build.sbt

```diff
@@ -42,7 +42,15 @@ libraryDependencies ++= {
         (scalaBinaryVersion in pluginCrossBuild).value
       )
     )
-  } else Nil
+  } else {
+    Seq(
+      Defaults.sbtPluginExtra(
+        "org.planet42" % "laika-sbt" % "0.7.5",
```

Comment:
Does the latest version only support sbt 1.0?

Reply:
Yep. Support for 0.13 has been completely dropped.